ord_schema/proto/reaction.proto

```diff
@@ -861,26 +852,104 @@ message ReactionProduct {
+    // using an isolated weight analysis, quantitative NMR with an internal
+    // standard, LC peak area with an internal standard and calibration curve
+    // obtained using an authentic standard. 
+    YIELD = 3;
+    // Selectivites include enantiomeric excess/ratios, diastereomic ratios,
+    // etc. The detailed type should be specified in selectivity_type.
+    SELECTIVITY = 4;
+    // The purity of the product mixture, normalized to all other species
+    // observed in the analysis.
+    PURITY = 5;
+    // Integrated peak area, e.g., as in an LC chromatogram.
+    AREA = 6;
+    // Retention time, e.g., as in an LC or GC chromatogram.
+    RETENTION_TIME = 7;
+    // Raw counts, e.g., as in an EIC or TIC from a MS analysis. If an EIC,
+    // the m/z ratio(s) should be defined in the details.
+    COUNTS = 8;
+    // Intensity, e.g., of a TLC spot, ELSD, or GC FID.
+    INTENSITY = 9;
+    // Direct m/z measurement, e.g., from HRMS. The value field should be used
+    // to report the m/z, not the "mass" field.
+    MZ = 10;
+    // Temperature, e.g., from a melting point experiment.
+    TEMPERATURE = 11;
+    // Angle, e.g., from a circular dichroism experiment. Units of degrees
+    // centimer squared per decimol are assumed.
+    ANGLE = 12;
   }
-  SelectivityType type = 1;
-  string details = 2;
-  optional float value = 3;
-  // TODO(kearnes): What does precision mean in this context?
-  optional float precision = 4;
+  MeasurementType type = 2;
+  // Additional information about the measurement and, in particular, how it
+  // was calculated if that is ambiguous given the analysis type.
+  string details = 3;
+  // Whether an internal standard was used for this measurement to normalize
+  // it, e.g., for quantification of yield or relative peak area.
+  optional bool uses_internal_standard = 4;
+  // Whether an authentic sample of a material was used for callibrating
+  // an analytical technique, e.g., for product identification by LCMS.
+  // "Authentic" refers to a sample of a compound whose structure has been
+  // proven by the accepted means (1H NMR, 13C NMR, IR, HRMS), and is now
+  // being used as a comparison for other analytical methods.
+  optional bool uses_authentic_standard = 5;
+  // Details about the authentic standard used, optionally including
+  // information about its precise salt form and source/vendor.
+  Compound authentic_standard = 6;
+  // The numeric value for this measurement. For example, the yield expressed
+  // as a percentage (i.e., 50% yield corresponds to 50 and not 0.5). If the
+  // value is associated with units, those unit-specific fields should be used
+  // instead.
+  optional float value = 7;
+  optional float precision = 8;
+  // The value for this measurement metric when it cannot be represented by a
+  // single number.
+  string string_value = 9;
+  Mass mass = 10;
+  Temperature temperature = 11;
+  Time time = 12;
```

Comment:
Do these refer to what temperature and time the measurement was taken at? Or, is it supposed to be used with analyses like MP and measurements like RETENTION_TIME that report a temperature or time?

Reply:
The latter -- I will add a comment to make that clearer

Reply:
Not completely relevant since we've removed structured analytical data from the ProductMeasurement. I've left in a retention_time field since I imagine that will be very commonly associated with an AREA or INTENSITY for LC or LCMS analyses. What do you think? This is still focusing on a product-specific measurement extracted from an analysis.

Would it make sense to add a repeated field for EIC so the m/z ratios used to get an INTENSITY are specified here? That seems like it might be a fairly common value to specify for HTE data. Knowing the EIC that was used to calculate. I've added one in, but let me know if you think this is extraneous.